components/Stats.vue

```diff
@@ -458,30 +417,12 @@ export default {
         ]
       }
     },
-    renameCountyNameForHospitalization(name) {
```

Comment:
We were using both these "short names" and the long ones, so I let the short names be the keys in the CountyData object and the long names be the "name" property on each county and changed the relevant references.